html/browsers/browsing-the-web/back-forward-cache/timers.html

```diff
@@ -0,0 +1,79 @@
+<!doctype html>
+<meta name="timeout" content="long">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/utils.js"></script>
+<script src="/common/dispatcher/dispatcher.js"></script>
+<script src="resources/helper.sub.js"></script>
+<script>
+// https://bugs.chromium.org/p/chromium/issues/detail?id=1202687
+// https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html#timers:fully-active
+promise_test(async t => {
+  const pageA = new RemoteContext(token());
+  const pageB = new RemoteContext(token());
+
+  const urlA = location.origin + executorPath + pageA.context_id;
+  const urlB = originCrossSite + executorPath + pageB.context_id;
+
+  const delayMain = 8000;
+  const delayBeforeNavigation = 3000;
+  const delayBeforeBackNavigation = 3000;
```

Comment:
These values do look rather long. I know debug builds can be very slow, but still these feel overly long.